blazor/speeddial/display-modes.md

```diff
@@ -7,17 +7,66 @@ control: SpeedDial
 documentation: ug
 ---
 
-# Display Modes in Blazor Speed Dial Component
+# Display Options in Blazor SpeedDial Component
 
 The action items in Blazor Speed Dial can be displayed in `Linear` and `Radial` display modes by setting [Mode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_Mode) property.
 
-## Linear display mode
+## Open trigger
+
+By default, action items are displayed only when clicking the speed dial button. You can use the [OpensOnHover](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_OpensOnHover) property to change the trigger that opens the action items to be on hover instead.
+
+{% tabs %}
+{% highlight razor %}
+
+@using Syncfusion.Blazor.Buttons
+
+<SfSpeedDial OpensOnHover=true Content="Edit">
+    <SpeedDialItems>
+        <SpeedDialItem IconCss="e-icons e-cut"/>
+        <SpeedDialItem IconCss="e-icons e-copy"/>
+        <SpeedDialItem IconCss="e-icons e-paste"/>
+    </SpeedDialItems>
+</SfSpeedDial>
+
+{% endhighlight %}
+{% endtabs %}
+
+![Blazor SpeedDial component with OpensOnHover](./images/Blazor-SpeedDial-OpensOnHover.png)
+
+## Animation
+
+The Speed Dial items can be animated during the opening and closing of the popup action items. You can customize the animation's `Effect`, `Delay`, and `Duration` by setting [SpeedDialAnimationSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SpeedDialAnimationSettings.html) tag directive. By default, Speed Dial animates with a `Fade` effect and supports all [SpeedDialAnimationEffect](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SpeedDialAnimationEffect.html) effects.
```

Comment:
By default, Speed Dial animates with a `Fade` effect and supports all the effects defines in [SpeedDialAnimationEffect]().

Reply:
Modified the content as given